renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:recommended"],
+  "dependencyDashboard": true,
+  "pinVersions": true,
+  "vulnerabilityAlerts": {
+    "groupName": "security updates",
+    "enabled": true,
+    "schedule": [],
+    "labels": ["security"],
+    "prCreation": "immediate"
+  },
+  "osvVulnerabilityAlerts": true,
+  "transitiveRemediation": true,
```

Comment:
I can't find that setting in the docs.

Reply:
I saw it in Marcel's link in the security example [here](https://github.com/renovatebot/renovate/discussions/15107#discussioncomment-5702811), and we need it to enable the automatic remediation of vulnerabilities found in transitive dependencies.

Reply:
The setting seems to have gotten removed https://github.com/renovatebot/renovate/blob/34a04c0b97033f713331e470f61b0e05f083ee95/lib/config/migrations/migrations-service.ts#L74